File: 165-class-average/solve.js
function getAverageGrade(scores) {
  if (!scores || scores.length === 0) {
    return null;
  }

  const scoresSum = scores.reduce((sum, score) => sum + score, 0);
  const averageScore = scoresSum / scores.length;

  if (averageScore >= 97) return "A+";
  else if (averageScore >= 93) return "A";
  else if (averageScore >= 90) return "A-";
  else if (averageScore >= 87) return "B+";
  else if (averageScore >= 83) return "B";
  else if (averageScore >= 80) return "B-";
  else if (averageScore >= 77) return "C+";
  else if (averageScore >= 73) return "C";
  else if (averageScore >= 70) return "C-";
  else if (averageScore >= 67) return "D+";
  else if (averageScore >= 63) return "D";
  else if (averageScore >= 60) return "D-";
  else return "F";
}
    

File: 165-class-average/solve.py
def get_average_grade(scores):
    if not scores:
        return None 

    scores_sum = sum(scores)
    average_score = scores_sum / len(scores)

    if average_score >= 97:
        return "A+"
    elif average_score >= 93:
        return "A"
    elif average_score >= 90:
        return "A-"
    elif average_score >= 87:
        return "B+"
    elif average_score >= 83:
        return "B"
    elif average_score >= 80:
        return "B-"
    elif average_score >= 77:
        return "C+"
    elif average_score >= 73:
        return "C"
    elif average_score >= 70:
        return "C-"
    elif average_score >= 67:
        return "D+"
    elif average_score >= 63:
        return "D"
    elif average_score >= 60:
        return "D-"
    else:
        return "F"
    